src/main/webapp/config.css

```diff
@@ -0,0 +1,84 @@
+.hidden {
```

Comment:
Is this one being used anywhere?

Reply:
sure is `<div name="sites" id="siteData_${loop.index}" class="hidden">`